### PR TITLE
wasting treatment outline

### PR DESCRIPTION
**Pull request by Ninicorn**

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]
Good start!  It is so important that we do these documents, so thank you for getting this one going.

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+  And then add it to the list of decorators above.
+
+
+===============================================
+Treatment and management for acute malnutrition 
+===============================================
+
+This documentation focuses on treatment and management of acute malnutrition in Ethiopia based on 2019 National Guideline for the Management of Acute Malnutrition. [EMOH]_
+
+The prevalence of stunting in Ethiopia has declined from 58% in 2000 to 38% in 2016. However, the prevalence of wasting has remained fairly static at 12% in 2000 and 10% in 2016. To address malnutrition in all its forms, the Government is applying two programmatic approaches. The first focuses on increasing access to and availability of food through improved economic growth, better agricultural production systems along with promotion of good nutrition practices and prevention of malnutrition. The second approach aims to strengthen early warning systems and timely emergency response, including wide-scale delivery of services for the management of acute
```
Comment (beatrixkh) [inline docs/source/intervention_models/wasting_treatment/index.rst:38]: Can you add citations here?

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    - Integrated Management of Newborn and Childhood illness
+
+
+Intervention Overview
+---------------------
+
+This flow chart summarizes the core aspects of the care and treatment for SAM and MAM, and integration into the
+routine health system. 
+
+.. image:: flow_chart_management_of_acute_malnutrition
```
Comment (beatrixkh) [inline docs/source/intervention_models/wasting_treatment/index.rst:73]: can you add definitions for sc, otp, and tsfp?

File: docs/source/intervention_models/wasting_treatment/index.rst [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+
+With medical complications
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Without medical complications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. todo::
+
+  Fill out the following table with a list of known outcomes affected by the intervention, regardless of if they will be included in the simulation model or not, as it is important to recognize potential unmodeled effects of the intervention and note them as limitations as applicable.
```
Comment (beatrixkh) [inline docs/source/intervention_models/wasting_treatment/index.rst:124]: This sounds great

**Review by beatrixkh — APPROVED** [review]
Thank you for getting this outline up!

File: docs/source/intervention_models/wasting_treatment/index.rst (hunk 1 above)
Comment (Ninicorn) [reply docs/source/intervention_models/wasting_treatment/index.rst:38]: I thought I did...I'll make sure it gets rendered!!! I'll finish this up today :)

File: docs/source/intervention_models/wasting_treatment/index.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/intervention_models/wasting_treatment/index.rst:73]: yes!!

**Pull request merged**